library/RDSYS

```diff
@@ -1,5 +1,6 @@
 (DEFINE-FILE-INFO PACKAGE "INTERLISP" READTABLE "XCL" BASE 10 FORMAT XCCS)
```

Comment:
RDSYS should use READTABLE "INTERLISP". That was one of the comments on the previous commits.